src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$white--blue--color: #00acdc;
+$dark--blue--color: #060b35;
+$white--color: #fff;
+$white--grey--color: #f3f3f3;
+$dark--grey--color: #616070;
+$blue--color: #34568b;
```

Comment:
Variable names should show their purpose. If the color palette changes and all elements that uses blue color will use red instead you have to find and rename in all places